pkg/monitor/monitor/monitor_manager.go

```diff
@@ -75,12 +75,24 @@ func (m *MonitorManager) SyncMonitor(monitor *v1alpha1.TidbMonitor) error {
 	}
 
 	var firstTc *v1alpha1.TidbCluster
+	assetStore := NewStore(m.deps.ConfigMapLister, m.deps.SecretLister)
+
 	for _, tcRef := range monitor.Spec.Clusters {
 		tc, err := m.deps.TiDBClusterLister.TidbClusters(tcRef.Namespace).Get(tcRef.Name)
 		if err != nil {
 			rerr := fmt.Errorf("get tm[%s/%s]'s target tc[%s/%s] failed, err: %v", monitor.Namespace, monitor.Name, tcRef.Namespace, tcRef.Name, err)
 			return rerr
 		}
+
+		// If cluster enable tls
+		if tc.IsTLSClusterEnabled() {
+			tcTlsSecretName := util.ClusterClientTLSSecretName(tc.Name)
+			err := assetStore.addTLSAssets(tc.Namespace, tcTlsSecretName)
+			if err != nil {
+				return err
+			}
+		}
+
 		if firstTc == nil && !tc.IsHeterogeneous() {
 			firstTc = tc
```

Comment:
Is this still necessary?

Reply:
How are the TidbCluster related envs defined in this function https://github.com/pingcap/tidb-operator/blob/master/pkg/monitor/monitor/util.go#L274 used now?

Reply:
After discussion, i remove tc from configuraiton generate process firstly.  

syncTidbMonitorConfig(tc *v1alpha1.TCCluster, dc *v1alpha1.DMCluster, monitor *v1alpha1.TidbMonitor)
simple to below method:
syncTidbMonitorConfig(dc *v1alpha1.DMCluster, monitor *v1alpha1.TidbMonitor)